events/models.py

```diff
@@ -3,31 +3,48 @@
 
 
 class Event(models.Model):
-    event = models.CharField(max_length=100, blank=True)
-    description = models.CharField(max_length=1000, blank=True)
-    start_time = models.DateTimeField()
-    end_time = models.DateTimeField()
+    priority = models.IntegerField(blank=True)
+    event = models.CharField(max_length=100, blank=False)
+    description = models.CharField(max_length=2000, blank=True)
     prize = models.CharField(max_length=20, blank=True)
-    venue = models.CharField(max_length=10, blank=True)
-    image = models.ImageField(upload_to='event-images/', null=True, blank=True)
+    team_size = models.CharField(max_length=20, blank=True)
+    venue = models.CharField(max_length=50, blank=True)
+    start_time = models.CharField(max_length=100, blank=False)
+    end_time = models.CharField(max_length=100, blank=False)
+    about = models.CharField(max_length=1500, blank=True)
+    pdf = models.URLField(max_length=500, null=True, blank=True)
 
     def __str__(self):
         return self.event
 
 
 class Contact(models.Model):
     event = models.ForeignKey(Event, on_delete=models.CASCADE)
-    name = models.CharField(max_length=100)
-    role = models.CharField(max_length=40)
-    phone_number = PhoneNumberField()
+    convenor = models.CharField(max_length=100, blank=False)
+    role1 = models.CharField(max_length=40)
+    phone_number1 = PhoneNumberField(blank=True)
+
+    co_convenor1 = models.CharField(max_length=100, blank=False)
+    role2 = models.CharField(max_length=40)
+    phone_number2 = PhoneNumberField(blank=True)
+
+    co_convenor2 = models.CharField(max_length=100, blank=True)
+    role3 = models.CharField(max_length=40, blank=True)
+    phone_number3 = PhoneNumberField(blank=True)
+
+    member1 = models.CharField(max_length=100, blank=False)
+    role4 = models.CharField(max_length=40)
+
+    member2 = models.CharField(max_length=100, blank=False)
+    role5 = models.CharField(max_length=40)
```

Comment:
We can make a separate model as `EventMember` and create as much the event will require?
It can also contain a ChoiceField for the position: Convenor, Co-Convenor and Member to differentiate.
Would make this more dynamic. views @aashutoshrathi ?

Reply:
It was created according to the frontend team's suggestions.

Reply:
I think yes, we can.
But this PR has reached its time.
Any more changes here will make this PR not so atomic (which it already was).
So I think if there are any issues in this, we should create issues for them and merge this one.
@Monal5031

Reply:
Yup let's do that.
Can you create issues for these mentioned comments @yashshah2820 ?

Reply:
I will try